feature/orchestrator/src/main/java/com/simprints/feature/orchestrator/usecases/steps/BuildStepsUseCase.kt

```diff
@@ -230,4 +337,51 @@ internal class BuildStepsUseCase @Inject constructor(
                 ),
             )
         )
+
+    private fun determineFingerprintSDKs(
+        projectConfiguration: ProjectConfiguration,
+        ageGroup: AgeGroup?,
+    ): List<FingerprintConfiguration.BioSdk> {
+        val sdks = mutableListOf<FingerprintConfiguration.BioSdk>()
+
+        if (!projectConfiguration.isAgeRestricted()) {
+            projectConfiguration.fingerprint?.allowedSDKs?.let { sdks.addAll(it) }
+        } else {
+            ageGroup?.let {
+                if (projectConfiguration.fingerprint?.secugenSimMatcher?.allowedAgeRange?.contains(ageGroup) == true) {
+                    sdks.add(FingerprintConfiguration.BioSdk.SECUGEN_SIM_MATCHER)
+                }
+                if (projectConfiguration.fingerprint?.nec?.allowedAgeRange?.contains(ageGroup) == true) {
+                    sdks.add(FingerprintConfiguration.BioSdk.NEC)
+                }
+            }
+        }
```

Comment:
I don't think we can support multiple fingerprint SDKs being used for a reference currently as for example for the `PersonCreation` event we only have one `fingerprintReferenceId`

Reply:
I think this should be left to the configuration. Long term we are moving in the direction of supporting multiple SDKs in the same workflow. Also, there are discussion about removing the age selection screen which might force us to capture and search with multiple SDKs during identification for example. 
All in all to say - I prefer to have flexible code from the beginning so we can easily build upon it in the future.